services/pictureServer.js

```diff
@@ -85,6 +85,9 @@ const picItems = [
 module.exports = {
     getItemsList() {
         return picItems;
+    },
+    findOne(id) {
```

Comment:
Make this method like generator, it will help in future to migrate to using mongo